crates/rpc/rpc-types/src/eth/engine/payload.rs

```diff
@@ -397,14 +399,20 @@ impl PayloadStatusEnum {
 /// Various errors that can occur when validating a payload or forkchoice update.
 ///
 /// This is intended for the [PayloadStatusEnum::Invalid] variant.
-#[derive(Debug, Clone, PartialEq, Eq, thiserror::Error)]
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq, thiserror::Error)]
 pub enum PayloadValidationError {
     /// Thrown when a forkchoice update's head links to a previously rejected payload.
     #[error("links to previously rejected block")]
     LinksToRejectedPayload,
     /// Thrown when a new payload contains a wrong block number.
     #[error("invalid block number")]
     InvalidBlockNumber,
```

Comment:
these are fine because they mirror geth's error messages.

There are no special rules for these messages, however it makes sense to mirror what's already in use to be consistent with other clients.